engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -644,6 +644,15 @@ protected void serviceStart() throws Exception
       if (SecurityUtils.isStramWebSecurityEnabled()) {
```

Comment:
OK, I still don't understand why it can't be moved to SecurityUtils that already has few static methods to deal with security and why is it better to have it in the StreamingAppMasterService.java